### ISSUE TITLE
Helm upgrade failed: unable to decode "": json

### ISSUE THREAD

**Issue by mceronja** [fjogeleit/policy-reporter#158]
Hello,

I am running policy-reporter Helm chart 2.9.1 for a couple of days now.
Everything is working fine. Also I use it together with [Flux](https://fluxcd.io/).
Yesterday policy-reporter 2.9.2 Helm chart was released and I went for upgrade.
I checked the commits and values file and the upgrade seemed quite straightforward, just increment the version to 2.9.2.

But the upgrade failed and I keep getting:
`Helm upgrade failed: unable to decode "": json: cannot unmarshal number into Go struct field ObjectMeta.metadata.labels of type string`
I couldn't get any more details out.

Since these last commits were related to Grafana monitoring part I also added
```
monitoring:
...
  grafana:
    # required: namespace of your Grafana installation
    namespace: monitoring-system
    dashboards:
      # Enable the deployment of grafana dashboards
      enabled: true
      # Label to find dashboards using the k8s sidecar
      label: grafana_dashboard
      value: "1"
```
**value: "1"** to the **values.yaml** file but that also didn't work.
Regarding other options in **values.yaml** I don't have anything fancy. I have metrics, monitoring, kyverno plugin and Slack webhook. All other stuff is set to default.

**Comment by fjogeleit** [comment]
Hey thanks for your report. I will check this asap

**Comment by fjogeleit** [comment]
Hey @mceronja I just released 2.9.3 with the related fix. Thanks for reporting and sorry for the circumstances

**Comment by mceronja** [comment]
Hi @fjogeleit, upgrade successful, works like a charm.

Thanks for the quick fix!
